cmd/commands/lite.go

```diff
@@ -24,6 +24,7 @@ func NewCmdLite() *cobra.Command {
 	cmd.Flags().String("config", "", "Path to the configuration file")
 	cmd.Flags().String("host", "", "host to run the API on")
 	cmd.Flags().StringP("port", "p", "8288", "port to run the API on")
+	cmd.Flags().String("redis-url", "", "URL to an external Redis instance to use")
```

Comment:
Redis' own docs refer to this as the server URI, not URL, we should align those. From the CLI's `--help`:

```
  -u <uri>           Server URI.
 ```
 
We could make this option `redis-uri` and add clearer context:

```suggestion
	cmd.Flags().String("redis-uri", "", "External Redis server URI (Optional). Uses self-contained in-memory Redis if not provided.")
```